features/forced-dark-mode.js

```diff
@@ -0,0 +1,20 @@
+// Forced Dark Mode
+// By JaydenDev, 2022, MIT license
+// Forces dark mode on all scratch pages (forums and editor)
+// incompatible with other dark mode features
+
+// credit to: n/a
+// https://stackoverflow.com/questions/15505225/inject-css-stylesheet-as-string-using-javascript
+function addStyle(styleString) {
+    const style = document.createElement('style');
+    style.textContent = styleString;
+    document.head.append(style);
+}
+
+addStyle(`
+    * {
+        background-color: invert(100%);
+    }
+`)
```

Comment:
This part is now completely unnecessary. Here's what it looks like on the Scratch homepage:
<img width="1440" alt="Screenshot 2022-11-23 at 10 15 17 PM" src="https://user-images.githubusercontent.com/86856959/203686115-e4b239f8-fd60-4cf5-964c-9d7c245c6faf.png">

Reply:
@rgantzos changes implemented!

Reply:
Not yet, images are still inverted.

Reply:
@rgantzos Well there isn't necessarily much I can do about that other than not using the lazy method. Or I can just make a lazy method that's compatible with the lazy method and just invert the images a second time.

Reply:
you could do:
```
img {
   filter: invert(1);
}
```

Reply:
@rgantzos ok

Reply:
implemented!